src/WebJobs.Extensions.Http/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    internal static class Constants
+    {
+        public const string DefaultRoutePrefix = "api";
+        public const string AzureFunctionsHttpRouteDataKey = "MS_AzureFunctionsHttpRouteData";
```

Comment:
These request properties are a contract with Functions. I debated on whether I should change the names here to something not Functions specific, e.g. MS_AzureWebJobsHttpRouteData